dutyfree-chart/templates/ingress.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "dutyfree.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+
+  labels:
+    {{- include "dutyfree.labels" . | nindent 4 }}
+
+  annotations:
+    kubernetes.io/ingress.allow-http: {{ .Values.ingress.http.enabled | quote }}
+  {{- if .Values.certificate.enabled }}
+    networking.gke.io/managed-certificates: {{ .Values.certificate.name | quote }}
+  {{- end }}
+  {{- with .Values.ingress.annotations }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  backend:
+    serviceName: {{ $fullName }}
+    servicePort: {{ $svcPort }}
+{{- if .Values.ingress.tls }}
+  tls:
+  {{- range .Values.ingress.tls }}
+    - hosts:
+      {{- range .hosts }}
+        - {{ . | quote }}
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  # rules:
```

Comment:
I guess this can go out? 😁

Reply:
well, I was kinda trying to play with rule, in case whoever wants to run it will add different tls rules, but was still too much to add as a template, so I left it commented out.